### ISSUE TITLE
User is not able to reorder the Column  headers using single pointer mode of operation:A11y_Blob Container Table_Table_Dragging movements

### ISSUE THREAD

**Issue by v-jagansai** [microsoft/AzureStorageExplorer#7856]
### GitHub Tags:
#A11ySev3; #A11yTCS; #DesktopApp;  #WCAG4.1.3; #ScreenReader; #Win11; #A11yWCAG2.2; #FTP; #Azure Tools Storage Explorer; #SH_WCAG2.2-Azure Tools Storage Explorer_Win32_April2024; #Win32; #WCAG:2.5.7

### Environment Details:
Application Name: Azure Tools Storage Explorer
Application version:1.33.0(97)
Az copy Version: 10.23.0
Operating System: Windows 11 version 23H2.

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Navigate to Table present in right side of the storage explorer.
7. Verify that User is not able to reorder the Column  headers using single pointer mode of operation for the table.

### Actual Results:
User is not able to reorder the Column  headers using single pointer mode of operation

### Expected Results:
User should be able to reorder the Column headers using single pointer mode of operation.

### User Impact:
Functionality provided only with dragging will create difficulty for users using input devices like a trackball, head pointer, eye-gaze system to perform the operation.

### Attachment:
![Videooftableheaders-ezgif com-optimize](https://github.com/microsoft/AzureStorageExplorer/assets/109065315/03094343-91e5-42a9-9579-1587b070a39c)

**Comment by v-jagansai** [comment]
[Screen Recording (43).webm](https://github.com/microsoft/AzureStorageExplorer/assets/109065315/04b90f55-9e5c-4a61-be1d-59e1e1446a4d)

**Comment by craxal** [comment]
@v-jagansai This functionality is already available via a dialog. This can be done by right-clicking any header cell and selecting "Customize Columns..." Selected columns can then be reordered with the buttons.

![image](https://github.com/microsoft/AzureStorageExplorer/assets/3473026/bcb252e0-a7d8-4fa6-b829-7741a054471e)

![image](https://github.com/microsoft/AzureStorageExplorer/assets/3473026/beb0717d-7519-45af-9b7f-adc53db1e086)

**Comment by v-jagansai** [comment]
![Screen Recording (43)](https://github.com/microsoft/AzureStorageExplorer/assets/109065315/899b7c27-5c18-4fc1-8cb7-93e5aaea9ca4)
Here issue is not repro , We are able access the resize column settings using keyboard Hence the bug is closed
